components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/OAuth2JWTTokenValidator.java

```diff
@@ -96,7 +99,15 @@ public boolean validateAccessToken(OAuth2TokenValidationMessageContext validatio
                 log.debug("Resolved tenant domain: " + tenantDomain + " to validate the JWT access token.");
             }
 
-            IdentityProvider identityProvider = getResidentIDPForIssuer(claimsSet.getIssuer(), tenantDomain);
+            String resourceResidentOrgId;
+            try {
+                resourceResidentOrgId = claimsSet.getStringClaim("org_id");
```

Comment:
Let's use a constant for "org_id"

Reply:
Resolved in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2086/commits/e5b792101b4251e33c6454ad4ae069aafdfe2c12